lua/entities/gmod_wire_egp/lib/egplib/parenting.lua

```diff
@@ -18,7 +18,7 @@ EGP.ParentingFuncs.addUV = addUV
 local function makeArray( v, fakepos )
 	local ret = {}
 	if isstring(v.verticesindex) then
-		if (not fakepos) then
+		if fakepos then
```

Comment:
Is this fixing a bug? Why is this 'if' inverted without any other changes?

Reply:
Believe this was meant to fix what #2781 did.